src/configmanager.h

```diff
@@ -103,6 +104,7 @@ class ConfigManager
 			MAX_MARKET_OFFERS_AT_A_TIME_PER_PLAYER,
 			EXP_FROM_PLAYERS_LEVEL_RANGE,
 			MAX_PACKETS_PER_SECOND,
+			COIN_PACKET_SIZE,
```

Comment:
You are missing this in your luascript.cpp file.
Please test the things you upload.